### PR TITLE
Fix settings docs

### PR DESCRIPTION
**Pull request by sisuresh**
The Settings upgrade docs are out of date

### CODE REVIEW

File: docs/validators/admin-guide/soroban-settings.mdx [hunk 1]
```diff
@@ -17,9 +17,13 @@ If you are being asked to vote for an upgrade, please move on to the [Examine a
...
+
 ### 1. Create an Upgrade Set
 
-The `stellar` CLI tool allows for the use of a JSON input file to encode a collection of Soroban settings into an base64-encoded string representing the XDR type `ConfigUpgradeSet`. You can use the [`pubnet_phase1.json`] and [`pubnet_phase2.json`] files as a starting point to formulate your upgrade proposal. Use the following command to create the required upgrade set XDR:
+The `stellar` CLI tool allows for the use of a JSON input file to encode a collection of Soroban settings into an base64-encoded string representing the XDR type `ConfigUpgradeSet`. You can use the [`pubnet_phase1.json`] and [`pubnet_phase2.json`] files as a starting point to formulate your upgrade proposal. You can also pull directly from a running core node using `curl -s "127.0.0.1:11626/sorobaninfo?format=upgrade_xdr" | stellar-xdr decode --type ConfigUpgradeSet --output json-formatted`. This will allow you to get the exact settings running on that core node in JSON format, making it easier to change only the value you want to. Once you have your JSON file with updated values,yUse the following command to create the required upgrade set XDR:
```
Comment (anupsdf) [inline docs/validators/admin-guide/soroban-settings.mdx:26]: typo: ,yUse

File: docs/validators/admin-guide/soroban-settings.mdx [hunk 2]
```diff
@@ -102,25 +111,26 @@ If you didn't provide the `--signtxs` parameter to the `get-settings-upgrade-txs
...
 ```
 
 ### 5. Schedule the Upgrades
 
-Now you can schedule the upgrade on all the required validators using stellar-core's `upgrades` command, providing the `ConfigUpgradeSetKey` output from line 14 above and an agreed upon time in the future:
+Now you can schedule the upgrade on all the required validators using stellar-core's `upgrades` command, providing the `ConfigUpgradeSetKey` output from line 9 above and an agreed upon time in the future:
 
 ```bash
-curl -G 'http://localhost:11626/upgrades?mode=set&upgradetime=YYYY-MM-DDTHH:MM:SSZ' --data-urlencode 'configupgradesetkey=<LINE_14_OUTPUT>'
+curl -G 'http://localhost:11626/upgrades?mode=set&upgradetime=YYYY-MM-DDTHH:MM:SSZ' --data-urlencode 'configupgradesetkey=<LINE_9_OUTPUT>'
```
Comment (anupsdf) [inline docs/validators/admin-guide/soroban-settings.mdx:133]: How about suggesting `http-command` instead of `curl` to avoid encoding issues?

Comment (sisuresh) [reply docs/validators/admin-guide/soroban-settings.mdx]: Done

File: docs/validators/admin-guide/soroban-settings.mdx (hunk 1 above)
Comment (sisuresh) [reply docs/validators/admin-guide/soroban-settings.mdx]: Fixed

**Review by anupsdf — APPROVED** [review]

**Pull request merged**